core/src/main/java/hudson/model/Run.java

```diff
@@ -1469,6 +1479,14 @@ public String toString() {
     	return new ByteArrayInputStream(charset != null ? message.getBytes(charset) : message.getBytes());
     }
    
+    /**
+     * Returns a Reader that reads from the log file.
+     *
+     * This is the simplest way to access the job's console log.
```

Comment:
Well…not necessarily. Depends on what you wanted to accomplish. For example, this will contain raw console notes, which might be unwanted in some contexts. `getLogText` is the most _general_ way to access the log, as you can use that to work with the log in a variety of ways.

Reply:
Agreed, we should not be doing arguable suggestions, especially if we return to JEP-207

```suggestion
```